pkg/windows/calico.go

```diff
@@ -7,23 +7,20 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"html/template"
```

Comment:
Does "html/template" provide you specific html functionality you're using? If not, it'd be preferred to use "text/template".

Reply:
Not really. I was just reusing that import (moved the templating stuff to calico.go). Thanks, I'll change that